engine/access/rpc/engine.go

```diff
@@ -171,6 +172,12 @@ func NewBuilder(log zerolog.Logger,
 		MaxMsgSize:                config.MaxMsgSize,
 		AccessMetrics:             accessMetrics,
 		Log:                       log,
+		CircuitBreakerConfig:      config.CircuitBreakerConfig,
+	}
+
+	circuitBreakerEnabled := false
+	if config.CircuitBreakerConfig != nil {
```

Comment:
rather than a fallback, I'm in favor of requiring that a `config.CircuitBreakerConfig` is provided, and adding a default value into the node builders